hyde/layout/job.j2

```diff
@@ -1,4 +1,14 @@
 {% extends "base.j2" %}
+{% from "macros.j2" import render_excerpt with context  %}
+
+{%  block endhead %}
+    <meta property="og:site_name" content="The Free Python Job Board"/>
+    <meta property="og:title" content="{{ resource.meta.company }}: {{ resource.meta.title }}"/>
+    <meta property="og:type" content="website"/>
```

Comment:
Looking at the documentation, I wonder if `article` might be better. If people share these the individual job listing is more important than the pythonjobs.github.io website.